src/schema/city/index.ts

```diff
@@ -82,7 +82,7 @@ const CityType = new GraphQLObjectType<any, ResolverContext>({
           max_distance: LOCAL_DISCOVERY_RADIUS_KM,
           has_location: true,
           at_a_fair: false,
-          partner_types: args.partnerType,
+          ...(args.partnerType ? { partner_types: args.partnerType } : {}),
```

Comment:
FYI `{ ...undefined }` works fine too and would eliminate the need for the empty object returned from your ternary.

Reply:
Oh I like that better, good idea.

Reply:
new to me also

Reply:
Since `falsy && whatever` returns `undefined`, I used this:

```ts
 ...(args.partnerType && { partner_types: args.partnerType })
```

Which is just 😚👌

Reply:
🔥

Reply:
PS @ashfurrow I’m sure this is what you meant, but just in case: `T && whatever` returns `T`, where `T` is a falsy value. I.e. it will only return `undefined` when `T` is `undefined`.